index.html

```diff
@@ -13,7 +13,37 @@
 </head>
 
 <body>
-    <p>Some Text</p>
+
+    <header class="page-header">
+
+        <div class="nav-bar">
+            <img src="images/logo.png" alt="logo">
+            <div class="meniu">
+                <nav class="frames">
+                    <ol>
+                        <li><a href="#">Home</a></li>
+                        <li><a href="#">Explore</a></li>
+                        <li><a href="#">About</a></li>
+                    </ol>
+                </nav>
+                <button class="button">Sign In</button>
+            </div>
+
+        </div>
+
+    </header>
+    <section class="section">
+        <div class="presentation">
+            <div class="title">
+                <p>Discover your next place to travel in Taiwan</p>
+            </div>
+            <div class="description">
```

Comment:
šis divas save viduje turi tik vieną elementą. Tokiu atveju nebūtina jo naudoti, klasę description galima dėti tiesiai ant `p`.